README.md

```diff
@@ -79,6 +88,12 @@ The elevator is a simple device, that has three variables `lightSwitch`, `floorN
 - Modbus
   - JavaScript
 
+## Current Mashups
+
+### Smart Home Mashup
+
+See the mashup's [readme](./mashups//smart-home/README.md).
```

Comment:
Is the double forward slash correct?

Reply:
Fixed [here](https://github.com/eclipse-thingweb/test-things/pull/39/commits/28d3fca73fba88444fdfd95ec9cb9aa3a08ac693).